src/herder/Upgrades.cpp

```diff
@@ -403,6 +552,25 @@ Upgrades::isValidForApply(UpgradeType const& opaqueUpgrade,
                                         ProtocolVersion::V_18) &&
               (upgrade.newFlags() & ~MASK_LEDGER_HEADER_FLAGS) == 0;
         break;
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+    case LEDGER_UPGRADE_CONFIG:
+    {
+        auto cfgUpgrade =
```

Comment:
missing `SOROBAN_PROTOCOL_VERSION` check?

Reply:
I thought the plan was to add the version checks in once we remove `ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION`. Added the check anyways so we don't forget.